tests/data/expected/main/jsonschema/null_and_array_v2.py

```diff
@@ -10,7 +10,7 @@
 
 
 class MyObjItem(BaseModel):
-    items: Optional[List[Any]] = None
+    items: Optional[List[Any]]
```

Comment:
I think you are right. I have pushed some changes to preserve this behaviour.

However I think this should be fixed in the next breaking release as this is clearly a bug.
`MyObjItem.items` is marked as required in the jsonschema. Also, now we have a discrepancy between the behaviour of the pydantic v1 and v2 models we generate because the v1 one (correctly) doesn't have the default None.